corehq/privileges.py

```diff
@@ -99,4 +99,13 @@ def get_name_from_privilege(cls, privilege):
             REPORT_BUILDER: _('User Configurable Report Builder'),
             TEMPLATED_INTENTS: _('Templated Intents'),
             CUSTOM_INTENTS: _('Custom Intents'),
-        }.get(privilege, privilege)
+            DATA_CLEANUP: _('Data Management'),
+        }
+        if privilege in titles:
+            return titles[privilege]
+        else:
+            from corehq.util.soft_assert import soft_assert
+            _assert = soft_assert(notify_admins=True)
+            _assert(False, '{} does not have a title defined, users are '
+                           'seeing the slug itself.'.format(privilege))
```

Comment:
i don't trust that all that much (I think because the webworkers restart all the time). if you're not sure i'd advocate for leaving this out since hopefully we'd find out pretty quickly if we were exposing bad text to the users